### ISSUE TITLE
Gurfurlur the Menacing does not agro and does not two hour.

### ISSUE THREAD

**Issue by Jessland86** [LegionDark/Issues#1639]
**Date & Time**: 
1803 EDT 9/28/2017

**FFXI Client Version (use `/ver`)**: 
30170905_1

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**: 
no

**Character Name**: 
Immortality

**Nation**: 
n/a

**Job(level)/Sub Job(level)**: 
n/a

**NPC or Monster or item Name**: 
Gurfurlur the Menacing

**Zone name**: 
Halvung

**Coordinates (use `$where`)**: 
-61 -23 11

**ffxiah.com link (for items issues only)**: 

**Multi-boxing? (multiple clients on same connection)**: 
n/a

**Steps To Reproduce / any other info**: 
Walk up to troll, wave at his face, he no agro you. He nice troll. Want you to just leave him alone.

He is also supposed to be able to use hundred fists multiple times, he does not appear to be using his two hour at all.

**Comment by TeoTwawki** [comment]
tl;dr
troll thinks he so much more BA than other trolls he snub players and refuse to use his best attack.

**Comment by Jessland86** [comment]
He's supposed to be: Aggressive and Link. And is supposed to detect by: True Sight, and Follow by Scent (oddly.) per http://ffxiclopedia.wikia.com/wiki/Gurfurlur_the_Menacing

**Comment by TeoTwawki** [comment]
Yes that is odd, you'd never think he could smell anything over his own stench!
